content/en/docs/Overview/_index.md

```diff
@@ -7,3 +7,40 @@ description: >
 ---
 
 Apache Parquet is a columnar storage format available to any project in the Hadoop ecosystem, regardless of the choice of data processing framework, data model or programming language.
+
+This documentation contains information about both the [parquet-mr](https://github.com/apache/parquet-mr) and [parquet-format](https://github.com/apache/parquet-format) projects. 
+
+
+### Parquet Format
+
+The "Parquet Format" project hosts the official specification of the Parquet file format, defining how data is structured and stored. This specification, along with Thrift metadata definitions and other crucial components, is essential for developers to effectively read and write Parquet files. The parquet-format project specifically contains the format specifications needed to understand and properly utilize Parquet files.
+
+As a repository focused on specification, the parquet-format repository does not contain source code. 
+
+
+### Parquet-MR 
+
+The parquet-mr GitHub repository is part of the Apache Parquet project and specifically focuses on providing Java tools for handling the Parquet file format within the Hadoop ecosystem. Essentially, this repository includes all the necessary Java libraries and modules that allow developers to read and write Parquet files.
+
+Parquet-MR can be seen as a "reference" implementation of parquet-format. There are a number of other Parquet Format implementations, which are listed below. 
+
+Included in parquet-mr:
+* Java/Scala Implementation: It contains the core Java/Scala implementation of the Parquet format, making it possible to use Parquet files in Java applications, particularly those based on Hadoop.
```

Comment:
Is there actually some Scala code in it?

Reply:
There is: https://github.com/apache/parquet-mr/blob/408c18b38e15ebca56b423687902440cf421768b/parquet-scala/src/main/scala/org/apache/parquet/filter2/dsl/Dsl.scala

Reply:
Perhaps we should just say Java implementation here. The scala code is just for filters and we don't have a full scala implementation.